docs/references/protocol/transactions/types/mptokenissuanceset.md

```diff
@@ -39,4 +40,14 @@ Transactions of the `MPTokenIssuanceSet` type support additional values in the `
 | `tfMPTLock`        | `0x00000001` | 1             | Enable to lock balances of this MPT issuance. |
 | `tfMPTUnlock`      | `0x00000002` | 2             | Enable to unlock balances of this MPT issuance. |
 
+## Error Cases
+
+Besides errors that can occur for all transactions, {% $frontmatter.seo.title %} transactions can result in the following [transaction result codes](../transaction-results/index.md):
+
+| Error Code                | Description |
+|:--------------------------|:------------|
+| `tecNO_PERMISSION`        | You are trying to lock or unlock an MPT, but the `tfMPTCanLock` flag wasn't set in the `MPTokenIssuanceCreate` transaction. | 
+| `temDISABLED`             | Either the `PermissionedDomains` or `SingleAssetVault` amendment is disabled. |
+| `temMALFORMED`            | You specified a `DomainID` and `Holder` value. You can only set one or the other in a single transaction. You can also receive this error if the transaction isn't changing anything; the transaction must either update a flag or modify the `DomainID`. |
```

Comment:
`tecOBJECT_NOT_FOUND` and `tecNO_DST` are examples of a couple other type-specific errors that can occur with this transaction, based on a quick skim of [the source](https://github.com/XRPLF/rippled/blob/afc05659edbc22d04b13402f157832778556f7b7/src/xrpld/app/tx/detail/MPTokenIssuanceSet.cpp#L138).